main.go

```diff
@@ -41,7 +41,7 @@ var (
 	localityZone          = flag.String("locality-zone", "", "the locality zone to use, instead of retrieving it from the metadata server. Useful when not running on GCP and/or for testing")
 	includeV3Features     = flag.Bool("include-v3-features-experimental", true, "whether or not to generate configs which works with the xDS v3 implementation in TD. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	includePSMSecurity    = flag.Bool("include-psm-security-experimental", true, "whether or not to generate config required for PSM security. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
-	secretsDir            = flag.String("secrets-dir-experimental", "/var/run/secrets/workload-spiffe-credentials", "path to a directory containing TLS certificates and keys required for PSM security. Used only if --include-psm-security-experimental is set. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
+	secretsDir            = flag.String("secrets-dir", "/var/run/secrets/workload-spiffe-credentials", "path to a directory containing TLS certificates and keys required for PSM security")
```

Comment:
nit: Granted that `include-psm-security-experimental` is true by default and is experimental and will be removed in future but this is still applicable only if `include-psm-security-experimental` is set to true.

Reply:
Yeah, but that flag wasn't being stabilized so it is very weird to call it out. I think it is fine to accept that someone setting `--include-psm-security-experimental=false` will figure out that `--secretsDir` no longer has an effect.